samples/features/sql-big-data-cluster/private-bdc/scripts/deploy-private-aks-udr.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+export SUBID= <your Azure subscription ID>
+
+export REGION_NAME=northeurope
+export RESOURCE_GROUP=private-bdc-rg
+export SUBNET_NAME=aks-subnet
+export VNET_NAME=bdc-vnet
+export AKS_NAME=bdcaksprivatecluster
+export FWNAME=bdcaksazfw
+export FWPUBIP=$FWNAME-ip
+export FWIPCONFIG_NAME=$FWNAME-config
+
+export FWROUTE_TABLE_NAME=bdcaks-rt
+export FWROUTE_NAME=bdcaksroute
+export FWROUTE_NAME_INTERNET=bdcaksrouteinet
+
+
+
+az group create -n $RESOURCE_GROUP -l $REGION_NAME
+ 
+az network vnet create \
```

Comment:
Please provide comments to each step, maybe with pointers to docs/references that will give better context to users to understand why that specific step is necessary.